default/scripting/buildings/ART_MOON.focs.txt

```diff
@@ -12,14 +12,16 @@ BuildingType
     ]
     enqueuelocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
-        [[SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS]]
```

Comment:
intentional? should be noted in commit message...

Reply:
It was consistent with Gaia and also with colony buildings, terraforming etc.
I can add it into the commit message

Reply:
I've had this problem before when working on terraforming. Now I think we should rather suggest to remove it from this from all temporary building.